src/spring/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+1.15.0
```

Comment:
Suggest using `upcoming` instead of `1.15.0`, as this version is not currently planned for immediate release

Reply:
updated version to upcoming